.github/workflows/python-k8s-test.yml

```diff
@@ -257,3 +275,10 @@ jobs:
         run: |
           terraform destroy -auto-approve \
             -var="test_id=${{ env.TESTING_ID }}"
+
+      - name: Delete deployment files
+        if: always()
+        continue-on-error: true
+        run: |
+          aws s3api delete-object --bucket aws-appsignals-sample-app-prod-${{ env.E2E_TEST_AWS_REGION }} --key python-frontend-service-depl-${{ env.TESTING_ID }}.yaml
+          aws s3api delete-object --bucket aws-appsignals-sample-app-prod-${{ env.E2E_TEST_AWS_REGION }} --key python-remote-service-depl-${{ env.TESTING_ID }}.yaml
```

Comment:
Going to need to add this to the cleanup script as well in case it's missed or it fails randomly

Reply:
Ok